assets/javascript/jquery.cookies.js

```diff
@@ -0,0 +1,154 @@
+"use strict";
+
+(function ($,undefined) {
```

Comment:
Why do we need this cookie logic?

Reply:
Cookie logic is to enable share buttons permanently.